### PR TITLE
Update annotation description in dynakube samples (#1402)

### PR DESCRIPTION
**Pull request by chrismuellner**
# Description
#1402

## Checklist
- [x] PR is labeled accordingly
- [x] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: assets/samples/applicationMonitoring.yaml [hunk 1]
```diff
@@ -190,10 +190,10 @@ spec:
     #
     # priorityClassName: priority-class
 
-    # Optional: Adds additional annotations to the ActiveGate pods
+    # Optional: Adds custom annotations to ActiveGate pods
     #
     # annotations:
-    #   annotation-key: annotation-value
+    #   custom: annotation
```
Comment (StefanHauth) [inline assets/samples/applicationMonitoring.yaml:196]:
```suggestion
    #   custom-key: annotation
```

File: assets/samples/classicFullStack.yaml [hunk 2]
```diff
@@ -113,22 +113,22 @@ spec:
       #
       # dnsPolicy: "ClusterFirstWithHostNet"
 
-      # Optional: Adds additional annotations to the OneAgent pods
+      # Optional: Adds custom annotations to OneAgent pods
       #
       # annotations:
-      #   annotation-key: annotation-value
+      #   custom: annotation
```
Comment (StefanHauth) [inline assets/samples/classicFullStack.yaml:119]:
```suggestion
      #   custom-key: annotation
```

File: assets/samples/classicFullStack.yaml [hunk 3]
```diff
@@ -228,12 +228,12 @@ spec:
     #
     # priorityClassName: priority-class
 
-    # Optional: Adds additional annotations to the ActiveGate pods
+    # Optional: Adds custom annotations to ActiveGate pods
     #
     # annotations:
-    #   annotation-key: annotation-value
+    #   custom: annotation
```
Comment (StefanHauth) [inline assets/samples/classicFullStack.yaml:234]:
```suggestion
    #   custom-key: annotation
```

File: assets/samples/cloudNativeFullStack.yaml [hunk 4]
```diff
@@ -124,12 +124,12 @@ spec:
       #
       # dnsPolicy: "ClusterFirstWithHostNet"
 
-      # Optional: Adds additional annotations to the OneAgent pods
+      # Optional: Adds custom annotations to OneAgent pods
       #
       # annotations:
-      #   annotation-key: annotation-value
+      #   custom: annotation
```
Comment (StefanHauth) [inline assets/samples/cloudNativeFullStack.yaml:130]:
```suggestion
      #   custom-key: annotation
```

File: assets/samples/cloudNativeFullStack.yaml [hunk 5]
```diff
@@ -257,10 +257,10 @@ spec:
     #
     # priorityClassName: priority-class
 
-    # Optional: Adds additional annotations to the ActiveGate pods
+    # Optional: Adds custom annotations to ActiveGate pods
     #
     # annotations:
-    #   annotation-key: annotation-value
+    #   custom: annotation
```
Comment (StefanHauth) [inline assets/samples/cloudNativeFullStack.yaml:263]:
```suggestion
    #   custom-key: annotation
```

File: assets/samples/hostMonitoring.yaml [hunk 6]
```diff
@@ -113,22 +113,22 @@ spec:
...
       #
       # labels:
       #   custom: label
 
-      # Optional: Adds additional annotations to the OneAgent pods
+      # Optional: Adds custom annotations to OneAgent pods
       #
       # annotations:
-      #   annotation-key: annotation-value
+      #   custom: annotations
```
Comment (StefanHauth) [inline assets/samples/hostMonitoring.yaml:125]:
```suggestion
      #   custom-key: annotations
```

File: assets/samples/hostMonitoring.yaml [hunk 7]
```diff
@@ -228,10 +228,10 @@ spec:
     #
     # priorityClassName: priority-class
 
-    # Optional: Adds additional annotations to the ActiveGate pods
+    # Optional: Adds custom annotations to ActiveGate pods
     #
     # annotations:
-    #   annotation-key: annotation-value
+    #   custom: annotation
```
Comment (StefanHauth) [inline assets/samples/hostMonitoring.yaml:234]:
```suggestion
    #   custom-key: annotation
```

File: assets/samples/hostMonitoring.yaml [hunk 8]
```diff
@@ -185,13 +185,13 @@ spec:
         cpu: 1000m
         memory: 1.5Gi
 
-    # Optional: Defines labels added to ActiveGate pods
+    # Optional: Adds custom labels to ActiveGate pods
     # Can be used to structure workloads
     #
     # labels:
     #   custom: label
```
Comment (StefanHauth) [inline assets/samples/hostMonitoring.yaml:192]:
```suggestion
    #   custom-key: label
```

File: assets/samples/hostMonitoring.yaml [hunk 9]
```diff
@@ -113,22 +113,22 @@ spec:
       #
       # dnsPolicy: "ClusterFirstWithHostNet"
 
-      # Optional: Defines labels added to OneAgent pods
+      # Optional: Adds custom labels to OneAgent pods
       # Can be used to structure workloads
       #
       # labels:
       #   custom: label
```
Comment (StefanHauth) [inline assets/samples/hostMonitoring.yaml:120]:
```suggestion
      #   custom-key: label
```

File: assets/samples/cloudNativeFullStack.yaml [hunk 10]
```diff
@@ -227,13 +227,13 @@ spec:
     #   key: node-role.kubernetes.io/master
     #   operator: Exists
 
-    # Optional: Defines labels added to ActiveGate pods
+    # Optional: Adds custom labels to ActiveGate pods
     # Can be used to structure workloads
     #
     # labels:
     #   custom: label
```
Comment (StefanHauth) [inline assets/samples/cloudNativeFullStack.yaml:234]:
```suggestion
    #   custom-key: label
```

File: assets/samples/classicFullStack.yaml [hunk 11]
```diff
@@ -185,13 +185,13 @@ spec:
         cpu: 1000m
         memory: 1.5Gi
 
-    # Optional: Defines labels added to ActiveGate pods
+    # Optional: Adds custom labels to ActiveGate pods
     # Can be used to structure workloads
     #
     # labels:
     #   custom: label
```
Comment (StefanHauth) [inline assets/samples/classicFullStack.yaml:192]:
```suggestion
    #   custom-key: label
```

File: assets/samples/classicFullStack.yaml [hunk 12]
```diff
@@ -113,22 +113,22 @@ spec:
...
       # annotations:
-      #   annotation-key: annotation-value
+      #   custom: annotation
 
-      # Optional: Defines labels added to OneAgent pods
+      # Optional: Adds custom labels to OneAgent pods
       # Can be used to structure workloads
       #
       # labels:
       #   custom: label
```
Comment (StefanHauth) [inline assets/samples/classicFullStack.yaml:125]:
```suggestion
      #   custom-key: label
```

File: assets/samples/applicationMonitoring.yaml [hunk 13]
```diff
@@ -147,13 +147,13 @@ spec:
         cpu: 1000m
         memory: 1.5Gi
 
-    # Optional: Defines labels added to ActiveGate pods
+    # Optional: Adds custom labels to ActiveGate pods
     # Can be used to structure workloads
     #
     # labels:
     #   custom: label
```
Comment (StefanHauth) [inline assets/samples/applicationMonitoring.yaml:154]:
```suggestion
    #   custom-key: label
```

**Review by StefanHauth — CHANGES_REQUESTED** [review]
The `custom-key` suggestions are probably only nitpicking, but the standalone use of the word `custom` feels a bit strange to me.

File: assets/samples/cloudNativeFullStack.yaml (hunk 5 above)
Comment (luhi-DT) [reply assets/samples/cloudNativeFullStack.yaml:263]: IMHO customers are mature enough to know that the first part of an annotation is always the key

**Pull request merged**